test/conformance/image/Makefile

```diff
@@ -33,14 +33,11 @@ CLUSTER_DIR?=$(shell pwd)/../../../cluster/
 
 # This is defined in root Makefile, but some build contexts do not refer to them
 KUBE_BASE_IMAGE_REGISTRY?=k8s.gcr.io
+BASE_IMAGE_VERSION?=buster-v1.9.0
+BASEIMAGE?=${KUBE_BASE_IMAGE_REGISTRY}/build-image/debian-base-${ARCH}:${BASE_IMAGE_VERSION}
 
-ifeq ($(ARCH),amd64)
-    BASEIMAGE?=${KUBE_BASE_IMAGE_REGISTRY}/build-image/debian-base:v2.1.3
-else
-    BASEIMAGE?=${KUBE_BASE_IMAGE_REGISTRY}/build-image/debian-base-${ARCH}:v2.1.3
-endif
-
-RUNNERIMAGE?=gcr.io/distroless/base:latest
+DISTROLESS_IMAGE?=base-debian10
```

Comment:
Do `BASEIMAGE` and `DISTROLESS_IMAGE` need to be based off the same distro of debian?  Like would the diff binary from buster be unhappy if there's a glibc from bullseye or vice-versa?

Reply:
Not 100% sure. But it makes sense to keep debian releases consistent across base and distroless like they are right now. Thinking of adding an inline comment to suggest that for future reference of the contributors. Let me know if that would suffice.

Reply:
A comment noting as much would probably be a good idea. I honestly don't know if there would be an issue either